### DISCUSSION TITLE
STOP worsening free plan!

### DISCUSSION THREAD

**Discussion by Usik77** [simple-login/app#2069] [Q&A] [answered]
Before you got bought by Proton you provided 15 aliases for free. Now 10.
You provided breach check for free. Now [it is paid](https://github.com/simple-login/app/pull/2065)
You provided send option for free. Now [it is paid](https://github.com/simple-login/app/commit/da09db3864a7bd72cef81f011f0805d26522a887)

But (!) allowed sending in trial period so this will not stop abuse because abuser will simply create new account.

The most confusing it that they make users use their service providing this for free but now they are taking it away. That is a pretty shameful practice. It should be paid from the beginning or free forever

They grabbed user base using this practice and now forcing users that came on previous conditions to pay because of new conditions

Shame on you!

**Comment by Schwest** [comment]
Agree. That’s pretty shady practice…

**Comment by nguyenkims** [comment, marked as answer]
Son, founder of SL here. 

> The most confusing it that they make users use their service providing this for free but now they are taking it away. That is a pretty shameful practice. It should be paid from the beginning or free forever
> They grabbed user base using this practice and now forcing users that came on previous conditions to pay because of new conditions

This is wrong, please do some research before posting incorrect info. Any new limit only affects new accounts. Accounts created before the enforcement date wasn't affected. 

This discussion is locked as it spreads false info.

1) The “free breach” is an unofficial feature that we haven’t advertised, as it was still in development. It uses the HIBP API that recently increases their price. And as the number of aliases in SL have significantly increased, we have to upgrade to the more expensive options in HIBP and financially can’t just propose this for free. The data breach monitoring feature is still under development, mostly for stability and scalability, we’ll make an announcement for it when it’s done.

2) The limit for creating reverse aliases has been mentioned on [SimpleLogin | Pricing](https://simplelogin.io/pricing/) since Apr 2022 ([commit](https://github.com/simple-login/website/commit/93326500cd127582d8279b2a1d845449a6bc2069)) before it was enforced last month. That leaves enough time for new users to be informed and make their choice. It only affects new accounts, existing accounts before the enforcement date aren’t affected and can still create new reverse aliases.
